test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -686,6 +692,23 @@ foo``u{2195}abc
             param ( $Script, $Expected )
             ExecuteCommand $Script | Should -Be $Expected
         }
+
+        $testInvalidNumerals = @(
+            @{ Script = "16p" }
+            @{ Script = "20ux" }
+            @{ Script = "18uu" }
+            @{ Script = "21ss" }
+            @{ Script = "100ll" }
+            @{ Script = "150su" }
+            @{ Script = "10ds" }
```

Comment:
Ah, good catch. Absolutely.

Reply:
@iSazonov Also noting down now before I forget:
Presently the literal `0x10d` is *legal* in PowerShell, although `Decimal.TryParse()` does *not* allow the hexadecimal syntax and that parse fails. Its actual return type is `Int32`. 

I plan on fixing this as I go through the parser rewrite after this PR.

Reply:
0x10d - have we tests for it?

Reply:
Oh, lol. I am clearly not *quite* up to speed this morning. Of *course* 0x10d is legal. D is a hexadecimal character... and that's exactly why decimals will never result from a hex literal ending in `d`. I'm gonna make a comment in the code that reflects this, because it's not immediately obvious why Decimal doesn't even bother to check for hex literals.

Reply:
(and yes I believe there are tests for it, but I'll pop one in the type checks just to be sure.)

Reply:
Added all these. :)